README.md

```diff
@@ -83,10 +83,18 @@ Then set the `YOURAPPLICATION_MODE` variable inside your shell, so the right set
 * 'TESTING': to execute the tests.
 * 'PRODUCTION': to run in production mode.
 
+Set your database. The following example show how to connect to your postgreSQL database `fhe_collector`.
+
+```
+export SQLALCHEMY_DATABASE_URI='postgresql://localhost/fhe_collector'
```

Comment:
is using env variables the common way to setup the postgres database?

Reply:
yes. so you can adapt it to your working environment. can also be set inside the setting-files.